content/200-concepts/100-components/drivers.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Drivers'
+metaTitle: 'Drivers (Preview)'
+metaDescription: 'Learn how you can use Prisma with other database drivers using driver adapters (Preview)'
```

Comment:
You are mixing two concepts here:
Drivers, and Driver Adapters.
Drivers are also used without Driver Adapters (native drivers for PostgreSQL, MySQL etc)

You need to untangle that.

Reply:
Updated! Let me know if it reads better